code/controllers/subsystem/starmap.dm

```diff
@@ -142,8 +142,10 @@ SUBSYSTEM_DEF(starmap)
 				current_planet = current_system.planets[1]
 			else if(in_transit_planet)
 				current_planet = to_planet
-			if(current_system.name == "Dolos") //Syndie cap
-				message_admins("The ship has just arrived at Dolos!")
+			if(istype(current_system,/datum/star_system/capital/syndicate)) //Syndie cap
```

Comment:
why is capital a subtype again.

anyways, giving systems a bitflag that can control specific behavior could be done, var/system_traits = DANGEROUS or something like that, would be less expensive but also allow us to do this for other things as well and would give more room for other special systems (nebulas when?)

Reply:
Capital systems have a custom generate() that doesn't bother with picking a location since their locations are hardcoded.

The check is 50% there bc jumping to Dolos is round over ~~and 50% I just really wanted to play music as the crew died~~

Reply:
titanic_flute.ogg

I like Qustinnus' idea. If you want specific systems to automatically toggle off admin logs, it should be it's own variable in the system.

The issue then becomes making sure admin logs are turned back on after the system is left.